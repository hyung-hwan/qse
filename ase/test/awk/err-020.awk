BEGIN {
        print abc > "123\0abc";
}

