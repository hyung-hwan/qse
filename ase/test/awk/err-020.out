BEGIN {
	print abc > "123\0abc";
}

RUN ERROR: CODE [115] LINE [2] i/o name containing a null character
